handlers.go

```diff
@@ -6,42 +6,74 @@ import (
 
 	"github.com/Financial-Times/go-fthealth/v1a"
 	"regexp"
+	"fleet/log"
+	"io"
+	"io/ioutil"
+	"encoding/json"
 )
 
 type httpHandlers struct {
 	baseURLMap map[string]string
+	vulcanAddr string
+	topic string
 }
 
 func (hh *httpHandlers) healthCheck() v1a.Check {
 	return v1a.Check{
-		BusinessImpact:   "Unable to write concepts to one or more of the configured writers",
-		Name:             "Check connectivity to neo rw services - services is a parameter in hieradata for this service",
+		BusinessImpact:   "Unable to connect to kakfa proxy",
+		Name:             "Check connectivity to kafka-proxy and presence of configured topic which is a parameter in hieradata for this service",
 		PanicGuide:       "TODO",
 		Severity:         1,
-		TechnicalSummary: `Cannot connect to Neo4j rw services. If this check fails, check that cluster is up and running and configured rw's are healthy. You can find the services configured as a parameter in hieradata for this service.`,
-		Checker:          hh.checker,
+		TechnicalSummary: `Cannot connect to kafka-proxy or configured topic is not present. If this check fails, check that cluster is up and running, proxy is healthy and configured topic is present on the queue.`,
+		Checker:          hh.checkCanConnectToProxy,
 	}
 }
 
 //TODO checks availability of writers not connectivity to Kafka?
```

Comment:
ok I thought healthcheck was to upstream services(proxy) and goodToGo was to downstream(writers) Ill add the proxy stuff into the goodToGo